File: CookingRecipes/Front-end/src/app/interfaces/measure.ts
export enum Measure {
  GRAM = 'GRAM',
  MILLILITRES = 'MILLILITRES',
  COUNT = 'COUNT'
}
export const MeasureMap: { [key in Measure]: string } = {
  [Measure.GRAM]: 'гр.',
  [Measure.MILLILITRES]: 'мл.',
  [Measure.COUNT]: 'бр.'
};

File: CookingRecipes/src/main/java/com/example/demo/enums/Measure.java
package com.example.demo.enums;

public enum Measure {
    GRAM("гр"),
    MILLILITRES("мл"),
    COUNT("бр");

    private final String measureName;

    Measure(String name) {
        this.measureName = name;
    }

    public String getMeasureName(){
        return measureName;
    }
}
